src/shrutiC/ValidateNumberIsFibonacci.java

```diff
@@ -0,0 +1,25 @@
+package shrutiC;
+
+public class ValidateNumberIsFibonacci {
+
+	void isNumInFibonacciSeries(int number) {
+		int num = 0, num1 = 1, sum = 0;
+		while (sum <= number) {
+			sum = num + num1;
+			num = num1;
+			num1 = sum;
+		}
+
+		if (number == num || number == num1 || number == sum)
```

Comment:
why these many conditions you require?
Check below code reference 

https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/1298/files